src/config/config.py

```
@@ -0,0 +1,34 @@
+"""設定管理モジュール"""
+from dataclasses import dataclass
+from typing import Optional
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)